docs/swagger/v1.json

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
No yaml?

Reply:
No.
We are following Azure Management REST APIs to use `json`. Is `yaml` preferred over `json`?

Reply:
I actually have no idea, I thought open API went to yaml instead of json.

Reply:
Both of them are okay according to [here](https://swagger.io/specification/#format).